packages/@aws-cdk/aws-appsync/lib/resolver.ts

```diff
@@ -112,6 +118,7 @@ export class Resolver extends Construct {
       requestMappingTemplate: props.requestMappingTemplate ? props.requestMappingTemplate.renderTemplate() : undefined,
       responseMappingTemplate: props.responseMappingTemplate ? props.responseMappingTemplate.renderTemplate() : undefined,
       cachingConfig: this.createCachingConfig(props.cachingConfig),
+      maxBatchSize: props.maxBatchSize,
```

Comment:
Should it not be `undefined` if it's not specified?

Reply:
Yes, will change it.

Reply:
If `props.maxBatchSize` isn't specified, it would be undefined with the code as is.  `maxBatchSize: props.maxBatchSize ? props.maxBatchSize : undefined;` is redundant, unless there's something I'm missing.

Reply:
@TheRealAmazonKendra 
You are probably right ( I am not too familiar with typescript). I think there is no drawback if it is checked. Actually it adds some consistency, since othere variables are checked aswell (line 115 dataSourceName).

Reply:
It's checked in line 115 because it's setting `dataSourceName` to `props.dataSource.name`. In this case, it would throw an error if `props.dataSource` doesn't exist. 

`dataSourceName: props.dataSource ? props.dataSource.name : undefined,`

That check is just redundant here, like it would be in lines 112-114.

Reply:
I have missed that. I will just revert it, removing the redundant call.